install.sh

```diff
@@ -34,7 +34,7 @@ then
 fi
 xcode_ver_num=`echo $xcode_ver | sed 's/^Xcode \([0-9][0-9]*\)\.\([0-9][0-9]*\)/\1\2/'`
 xcode_ver_str=`echo $xcode_ver | sed 's/^Xcode //'`
-if [[ $xcode_ver_num -lt $XCODE_MIN_VERSION ]]
```

Comment:
I forget how regexs differ between platform implementations.  The regex in line 35 should be:

        sed 's/^Xcode \([0-9][0-9]*\)\.\([0-9][0-9]*\).*$/\1\2/'